app/views/employees/_form.html.erb

```diff
@@ -1,5 +1,11 @@
 <%= simple_form_for @employee do |form| %>
-  <%= form.input :slack_username %>
+  <%= form.input :slack_username,
+                 required: true,
+                 pattern: '^[^@]\w+[^ ]$',
+                 input_html: {
+                   title: I18n.t('employees.errors.slack_username')
```

Comment:
can we write a feature spec that confirms that this shows up?

Also, did you think about using the default `simple_form` locales instead? https://github.com/plataformatec/simple_form#i18n

If you put the value in the right yml location, I believe you don't need to explicitly include it in the form.